timber/src/main/java/timber/log/Timber.java

```diff
@@ -446,13 +455,11 @@ protected String createStackElementTag(StackTraceElement element) {
      *
      * {@inheritDoc}
      */
-    @Override protected void log(int priority, String tag, String message, Throwable t) {
+    @Override protected void log(int priority, String tag, String message, Throwable e) {
+      String callerInfo = getCallerInfo(new Throwable().getStackTrace());
```

Comment:
I agree with you that this is a debug tool, but why should we slow down our debug build if we can use a bit fragile and dirty hack in preference to speed? We can steal stack trace from createStackElementTag or create it by ourselves if there is an explicit tag?